jeb_bearer.cfdg

```diff
@@ -0,0 +1,34 @@
+startshape LINES
+
+rule LINES {
+	LINE{r 45 hue 90 sat 1 brightness 0}
+	LINE{r -45 hue 0 sat 1 brightness 0 x 10}
+	LINE{r 45 hue 215 sat 1 brightness 0 x 5 y 3}
+}
+
+rule LINE {
+	SQUARE{}
+
+	// Increase z as we go so that when two lines overlap, the newer line gets
+	// drawn on top.
+	LINE{y 0.8 z 0.1}
```

Comment:
I didn't realize that using "z" could help you determine which objects to draw on top of each other. I always assumed that the code would be run from the first rule written to the last rule written, and so the first object rendered will always be below any object rendered after that.